omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -72,6 +72,7 @@
 default['supermarket']['install_directory'] = '/opt/supermarket'
 default['supermarket']['app_directory'] = "#{node['supermarket']['install_directory']}/embedded/service/supermarket"
 default['supermarket']['log_directory'] = '/var/log/supermarket'
+default['supermarket']['log_mode'] = '0700'
```

Comment:
We need to make sure this also gets modified in the docs. Those are in this same repo now so this can all be done in this PR

Reply:
sure, will update the same.

Reply:
doc updated for permission

Reply:
Should the setting be '700' or '0700'. Compare to docs-chef-io/content/supermarket/config_rb_supermarket.md